keyboards/mechlovin/infinity875/config.h

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021 Mechlovin' Studio
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID   0x4D4C // ML
+#define PRODUCT_ID  0x0875
+#define DEVICE_VER  0x0001
+#define PRODUCT     Infinity87.5
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 17
+ 
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { D5, D2, D4, D3, D0, D1 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION ROW2COL
+
+#define LED_CAPS_LOCK_PIN D6
+#define LED_SCROLL_LOCK_PIN B4
+#define LED_PIN_ON_STATE 1
+
+#ifdef RGB_MATRIX_ENABLE
+#define RGB_DI_PIN E2
+#define DRIVER_LED_TOTAL 42
+#define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+#define RGB_DISABLE_WHEN_USB_SUSPENDED false // turn off effects when suspended
```

Comment:
```suggestion
#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
```
Or if you don't want it to sleep at all: 

```suggestion
// #define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
```